db/migrate/20150618130443_enable_uuid_extension.rb

```diff
@@ -0,0 +1,5 @@
+class EnableUuidExtension < ActiveRecord::Migration
+  def change
+    enable_extension 'uuid-ossp'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.